src/types/node.rs

```diff
@@ -0,0 +1,6 @@
+//! Type of node (Parity, Geth...)
+
+pub enum NodeType {
```

Comment:
Would you mind adding documentation to all public types and variants in this file and `peer.rs`?

Reply:
added missing docs to both ``node.rs`` and ``peer.rs``.